l10nzhtw.properties

```diff
@@ -1,14 +1,20 @@
 category=Localization
 description=SonarQube Traditional Chinese Language Pack (not compatible with the "Chinese Pack" which uses Simplified Chinese)
-homepageUrl=https://github.com/timlee/sonar-l10n-zh-tw
+homepageUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw
 archivedVersions=
-publicVersions=1.0
+publicVersions=1.0,1.1
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
-defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+defaults.mavenArtifactId=sonar-l10n-zh-tw-plugin
+
+1.1.description=SonarQube Traditional Chinese Pack Support SonarQube 10.* Version
+1.1.sqVersions=[10.0,LATEST]
+1.1.date=2024-10-21
+1.1.changelogUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw/releases/tag/1.1
+1.1.downloadUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw/releases/download/1.1/sonar-l10n-zh-tw-plugin.jar
```

Comment:
This is 404

Reply:
Ah, sorry. I changed the jar file, so the old URL returns a 404 error.

Reply:
Done